packages/scss/src/components/_page-header.scss

```diff
@@ -0,0 +1,48 @@
+.page-header {
+	@include elevate(1);
+	padding: _theme('commons.padding');
+	padding-bottom: _theme('spacings.standard');
+	background-color: _color('white');
+
+	.menu {
+		&::after {
+			display: none;
+		}
+	}
+
+	&.mod-withBreadcrumbs {
```

Comment:
Can we suppose that all breadcrumbs inside a .page-header will have a margin-bottom 0 ? Like that we can delete the "mod-withBreadcrumbs"